client/components/LearnerViewDirective.js

```diff
@@ -45,6 +45,12 @@ tie.directive('learnerView', [function() {
                   {{paragraph.getContent()}}
                 </p>
               </div>
+              <div class="tie-dot-container"
+                   ng-class="{'tie-dot-hidden': hideDotDiv}">
```

Comment:
The reason was in case we add additional css animation. Since we don't have any right now, I changed it to ng-if, as suggested. I changed hideDotDiv to loadingIndicatorIsShown, as suggested.